test/loadtime/cmd/report/main.go

```diff
@@ -0,0 +1,86 @@
+package main
+
+import (
+	"encoding/csv"
+	"flag"
+	"fmt"
+	"log"
+	"os"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/tendermint/tendermint/store"
+	"github.com/tendermint/tendermint/test/loadtime/report"
+	dbm "github.com/tendermint/tm-db"
+)
+
+var (
+	db     = flag.String("database-type", "goleveldb", "the type of database holding the blockstore")
+	dir    = flag.String("data-dir", "~/.tendermint/data", "path to the directory containing the tendermint databases")
```

Comment:
Consider reading TMHOME when constructing the default value of "data-dir" (for consistency with Tendermint).

Reply:
TBH, I'd almost rather have no default for this flag. The reason I've used a flag here instead of using TMHOME is that the tool is, in the current iteration, unlikely to be used on the machine that the db was generated on. The thinking is that we'll zip up the `/data` directory and pull it down for analysis. If that's the targeted usecase, it doesn't really make sense to use the `TMHOME` since it's unlikely we'll actually be using the normal Tendermint data directory on a machine and doing so may accidentally rip data out of a test instance on a users's machine when they don't intend to.

Reply:
Yeah, I like the "no default" solution.